viewmaster/doc/viewmaster_v1.md

```
@@ -0,0 +1,257 @@
+   sudo -E python viewmaster/viewmaster.py
+   ```
+
+2. Open your browser and go to:
+   [http://127.0.0.1:8080/](http://127.0.0.1:8080/)
+   *(This corresponds to `VIEWMASTER_PREFIX_` in `viewmaster_config.py`.)*
+
+---
+
+# Key Components
+
+## 1. Main Application Code (`viewmaster/`)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)